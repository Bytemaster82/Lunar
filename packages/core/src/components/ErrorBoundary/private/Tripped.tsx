import React from 'react';
import T from '../../Translate';
import Alert from '../../Alert';

/* @ignore */
export default function Tripped() {
  return (
    <Alert
      danger
      title={
        <T
          phrase="This feature has crashed or failed to load."
          context="Loading a feature crashed"
        />
      }
    >
      <T phrase="Please try again." context="Message letting the user known to try again" />
    </Alert>
  );
}
