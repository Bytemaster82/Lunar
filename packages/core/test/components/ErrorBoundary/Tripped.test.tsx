import React from 'react';
import { shallow } from 'enzyme';
import Tripped from '../../../src/components/ErrorBoundary/private/Tripped';

describe('<Tripped />', () => {
  it('will render an alert with a message', () => {
    const wrapper = shallow(<Tripped />);

    expect(wrapper).toMatchSnapshot();
  });
});
