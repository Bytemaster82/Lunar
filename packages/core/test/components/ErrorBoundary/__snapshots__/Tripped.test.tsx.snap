// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tripped /> will render an alert with a message 1`] = `
<Alert
  danger={true}
  title={
    <Translate
      context="Loading a feature crashed"
      html={false}
      phrase="This feature has crashed or failed to load."
    />
  }
>
  <Translate
    context="Message letting the user known to try again"
    phrase="Please try again."
  />
</Alert>
`;
